src/ResourceManager/Network/Commands.Network/Commands.Network.csproj

```diff
@@ -801,6 +814,10 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\KeyVault\Commands.KeyVault\Commands.KeyVault.csproj">
```

Comment:
@akshaysngupta same comment about removing this reference

Reply:
removed.